src/app/recipebook-detail/recipebook-detail.component.html

```diff
@@ -1,38 +1,39 @@
-<app-nav-bar></app-nav-bar>
 
-<p>recipebook-detail works!</p>
-
-  <img src={{recipe.imageLink}}>
-
-<table>
-    <thead>
-      <th>Name</th>
-      <th>Description</th>
-      <th>Ingredients</th>
-      <th>Instructions</th>
-      <th>Cook Time</th>
-      <th>Difficulty</th>
-      <th>Serving Size</th>
-
-
-    </thead>
-    <tbody>
-      <tr>
-
-        <td>{{recipe.name}}</td>
-        <td>{{recipe.desc}}</td>
-        <td>{{recipe.ingredients}}</td>
-        <td>{{recipe.instructions}}</td>
-        <td>{{recipe.cookTime}}</td>
-        <td>{{recipe.difficulty}}</td>
-        <td>{{recipe.servingSize}}</td>
-     
-
-        <td>
-          <!-- Update to route to edit component and use recipe.id as a param -->
-            <a [routerLink]="['/savedrecipes/edit', recipe.id ]"><button>Edit</button></a>
-            <button (click)="deleteUserRecipe(recipe.id, user.uid)">Delete</button>
-        </td>
-      </tr>
-    </tbody> 
-  </table>
+ <!DOCTYPE html>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <app-nav-bar></app-nav-bar>
+  <br/>
+  <body class="bg">
+  <mat-card class="recipeBookDetails">
```

Comment:
Similar questions here as for the recipe detail page. Lets discuss these when you get the chance.